kibbeh/src/ui/SearchHistory.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import SearchIcon from "../icons/sm-solid-search.svg";
+
+export interface SearchHistoryProps {
+  onClickToDeleteSearchHistory: React.MouseEventHandler<HTMLSpanElement>
+  searchText: string
+}
+
+const SearchHistory: React.FC<SearchHistoryProps> = ({ onClickToDeleteSearchHistory, searchText }) => {
+  return (
+    <div className={"flex flex-row bg-primary-900"}>
+      <div className={"flex flex-1"}>
+        <SearchIcon className={"mr-2"} />
```

Comment:
Please add `items-center` class to the parent of the search icon and the span. Icon is slightly off-center.